gradle/libs.versions.toml

```diff
@@ -39,20 +39,35 @@ cucumber-java = { module = "io.cucumber:cucumber-java", version.ref = "cucumber"
 cucumber-spring = { module = "io.cucumber:cucumber-spring", version.ref = "cucumber" }
 cucumber-junit = { module = "io.cucumber:cucumber-junit", version.ref = "cucumber" }
 cucumber-junit-platform-engine = { module = "io.cucumber:cucumber-junit-platform-engine", version.ref = "cucumber" }
+junit = { module = "org.junit.jupiter:junit-jupiter" }
+junit-platform = { module = "org.junit.platform:junit-platform-suite" }
 
 [bundles]
 security = ["java-jwt", "spring-security"]
 jackson = ["jackson-core", "jackson-datatype-jsr310"]
 jpa = ["springBoot", "springBoot-jpa"]
 elasticsearch = ["spring-elasticsearch"]
 kafka = ["spring-kafka"]
-testing = ["assertj-core", "mockito-inline"]
+testing = ["assertj-core", "mockito-inline", "junit", "junit-platform"]
```

Comment:
The `junit-platform` dependency was added to `modernization-api` because Cucumber requires the JUnit Platform.  The libs projects use the Jupiter Platform which is provided through the `jvm-test-suite` plugin.

We might need to add a separate bundle for `junit` and `junit-platform` or maybe add those two to the existing `cucumber` bundle.